Sources/Leaf/Tag/Models/Equal.swift

```diff
@@ -0,0 +1,58 @@
+public final class Equal: BasicTag {
+    public enum Error: LeafError {
+        case expected2Arguments
```

Comment:
Non important tidbit: It's more grammatically correct to spell out numbers less than 10. http://www.quickanddirtytips.com/education/grammar/how-to-write-numbers